day05/haha/utils.js

```diff
@@ -0,0 +1,22 @@
+const panels = document.querySelectorAll(".panels div");
+
+panels.forEach((el) => {
+  // 監聽按下
+  el.addEventListener("click", clickToggle);
+});
+
+panels.forEach((el) => {
+  // 監聽動畫結束
+  el.addEventListener("transitionend", subtitleTransition);
+});
+
+// 展開收合 card ( toggle 自動切換)
+function clickToggle() {
+  this.classList.toggle("open");
```

Comment:
this event listener will trigger other element tag  
![image](https://user-images.githubusercontent.com/40877568/149552620-90fe8d26-ee61-4719-948e-0844f5c7510b.png)

Reply:
I suggest here neet to another ways to realize it
I will refactor it , thanks your good idea

Reply:
Here recommend two way .
JS: [closest](https://developer.mozilla.org/en-US/docs/Web/API/Element/closest) 

CSS: [pointer-events](https://developer.mozilla.org/en-US/docs/Web/CSS/pointer-events)

Reply:
I already use another way to refact it

Reply:
I don't know why you tag [pointer-events](https://developer.mozilla.org/en-US/docs/Web/CSS/pointer-events)  for me
This way can help me what?

Reply:
[closest](https://developer.mozilla.org/en-US/docs/Web/API/Element/closest) is a good another way to find node.
It can help me just one step replace closest with confirm nodeName to be current node or it's parent node.
I learned it thx!

Reply:
use `pointer-events: none;` the element trigger will ignored.

Reply:
ok I got it . In my new way is no longer trigger